frostsnap_core/src/device.rs

```diff
@@ -536,6 +569,7 @@ impl SignerState {
             SignerState::KeyGen { .. } => "KeyGen",
             SignerState::KeyGenAck { .. } => "KeyGenAck",
             SignerState::AwaitingSignAck { .. } => "AwaitingSignAck",
+            SignerState::DisplayBackup { .. } => "DisplayBackup",
```

Comment:
Something i noticed,  `DisplayBackup` is now a `SignerState` inside `frostsnap_core`. At some point we may want to separate the extra device functionality above a more general `frostsnap_core::Signer`. 

This would be useful for implementing other signers for our own purposes (on coordinator or on server). With this separation we could document a set of fundamental `CoordinatorToSignerMessage`s that can be implemented by compatible coordinators/hwws without the bells and whistles .

Reply:
Is displaying the backup an "extra functionality" though? It seems like core functionality. Remember that the concrete device impl can decide what to do when the coordinator instructs the device to display backup -- it can always deny the request for example.